infrastructure/monitoring/token_decorator.py

```
@@ -0,0 +1,291 @@
+                        if usage:
+                            input_tokens = getattr(usage, 'prompt_tokens', 0) or 0
+                            output_tokens = getattr(usage, 'completion_tokens', 0) or 0
+
+                    if len(response_chunks) < 10:
+                        try:
+                            if isinstance(chunk, str):
+                                response_chunks.append(chunk)
+                            elif hasattr(chunk, '__dict__'):
+                                response_chunks.append({k: v for k, v in vars(chunk).items() if not k.startswith('_')})
+                        except Exception:
+                            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)